solidus_easypost.gemspec

```diff
@@ -1,8 +1,12 @@
 # encoding: UTF-8
+lib = File.expand_path('../lib/', __FILE__)
```

Comment:
I think you can just use `require_relative` here, [as it's done in Solidus](https://github.com/solidusio/solidus/blob/4a1e245f2f8d300de45bf4efc2b6a425b58f2b5c/solidus.gemspec#L3).

Reply:
much cleaner indeed. fixed :)